src/content/en/2022/webassembly.md

```diff
@@ -1,22 +1,160 @@
 ---
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: WebAssembly
-#TODO - Review and update chapter description
-description: WebAssembly chapter of the 2022 Web Almanac covering Wasm compression, section sizes, most popular instructions, and post-MVP proposals
+description: WebAssembly chapter of the 2022 Web Almanac covering usage, languages, and post-MVP features.
 authors: [ColinEberhardt]
 reviewers: [binji, RReverser]
 analysts: [JamieWhitMac]
-editors: []
+editors: [tunetheweb]
 translators: []
+ColinEberhardt_bio: Colin is Technology Director at <a hreflang="en" href="https://www.scottlogic.com/">Scott Logic</a> and is a prolific technical author, blogger and speaker on a range of technologies. He is a board member of <a hreflang="en" href="https://www.finos.org/">FINOS</a>, which is encouraging open source collaboration in the financial sector. He is also very active on GitHub, contributing to a number of different projects.
```

Comment:
@ColinEberhardt I got this bio from your blog. Feel free to edit to something else.